docs/mine/lotus/sector-pledging.md

```diff
@@ -28,7 +28,7 @@ To pledge a sector use:
 lotus-miner sectors pledge
 ```
 
-This will, by default, pledge the space for 546 days.
+This will, by default, pledge the space for 540 days.
```

Comment:
@rjan90 hmmm... I think neither of these numbers is correct.

It *looks* like we do:
https://github.com/filecoin-project/lotus/blob/v1.10.0-rc1/storage/miner.go#L159-L160
which is `(540 * builtin.EpochsInDay) - ( builtin.EpochsInDay * 2 )` == `538` days / `1549440` epochs

@rjan90 can you confirm experimentally this is the case?

Reply:
hmmmm... tested with a CC-sector just now.
Started pledge command at: 827851
Finished AP at: 827853
PC2 finished at: 828233
C2 finished at: 828450

Expiration at: 2379243

Reply:
thats 538.46.......... days

you have the epoch the commit message hit the chain?

Reply:
@f8-ptrk 828450 [https://filfox.info/en/message/bafy2bzacedjvzrkmzvztkdehj5yermdnwkcnqgud6ceph7urxdq2oumynnhlu](https://filfox.info/en/message/bafy2bzacedjvzrkmzvztkdehj5yermdnwkcnqgud6ceph7urxdq2oumynnhlu)

Reply:
> @rjan90 hmmm... I think neither of these numbers is correct.
> 
> It _looks_ like we do:
> https://github.com/filecoin-project/lotus/blob/v1.10.0-rc1/storage/miner.go#L159-L160
> which is `(540 * builtin.EpochsInDay) - ( builtin.EpochsInDay * 2 )` == `538` days / `1549440` epochs
> 
> @rjan90 can you confirm experimentally this is the case?

this is a tech debt in lotus https://github.com/filecoin-project/lotus/issues/4760 =. will fix it. that being said, the value in actors should be the source of truth, 180-540 days in the epoch it is

Reply:
```suggestion
By default, this will pledge the space for 180 to 540 days.
```

Are we all happy with this?

Reply:
I do not think this is good...  We should adjust the docs when we actually implement the tech debt - it will likely be multiple months before then.

@jennijuju deferring to you though.

Reply:
`By default, this will pledge the space for 180 to 540 days.`

if you this please make clear under what circumstances it is less than 540! it confuses newcomers. it needs to be clear that CC sectors are always 540 days

Reply:
@jennijuju made issue #888 to add min-max values. I'll close this PR.